### ISSUE TITLE
Failing test depending on the seed

### ISSUE THREAD

**Issue by othmanesebbouh** [ott-jax/ott#302]
**Describe the bug**
The following test for fused GW fails when changing the seed.
https://github.com/ott-jax/ott/blob/ab9d89f1c9e2449601b3664c91f8568e062597fc/tests/solvers/quadratic/fgw_test.py#L250
We encountered this when transitioning from passing seed to passing PRNG keys to the functions (PR #290).

**To Reproduce**
Redefining the function (outside of the class) and using `test_fgw_lr_generic_cost_matrix(rng=jax.random.PRNGKey(0), cost_rank=(2, 3, 4))` results in an `AssertionError` at 
```python 
assert out.converged
```

**Expected behavior**
On a toy example with a high convergence tolerance, the algorithm should converge regardless of the seed.

**Comment by gjhuizing** [comment]
Hi, is this still an issue?

Noticed that when replacing `jax.random.uniform` with  `jax.random.normal` in https://ott-jax.readthedocs.io/en/latest/tutorials/GWLRSinkhorn.html FGW and LRFGW do not converge.

**Comment by michalk8** [comment]
Closing this due to inactivity.
